eng/scripts/Migrate-DevOps-Release-WorkItems.ps1

```diff
@@ -0,0 +1,248 @@
+# Script to migrate Java package work items in Azure DevOps
+# Updates Package field from artifact name to groupId+artifactName format
+
+param (
+  [Parameter(Mandatory = $false)]
+  [string]$DevOpsOrganization = "azure-sdk",
+  [Parameter(Mandatory = $false)]
+  [switch]$DryRun = $false
+)
+
+# Import DevOps work item helper functions
+. (Join-Path $PSScriptRoot ".." "common" "scripts" "Helpers" "DevOps-WorkItem-Helpers.ps1")
+
+<#
+.SYNOPSIS
+Find all Java package work items in Azure DevOps
+
+.DESCRIPTION
+Fetches all work items for Java language from Azure DevOps release tracking.
+Custom implementation that doesn't filter by Package field to find all Java items.
+
+.PARAMETER DevOpsOrganization
+The Azure DevOps organization name. Default: azure-sdk
+
+.PARAMETER IncludeClosed
+Include closed work items in the results. Default: false
+
+.OUTPUTS
+Array of work items for Java packages
+
+.EXAMPLE
+$javaWorkItems = Find-JavaPackageWorkItem -DevOpsOrganization "azure-sdk" -DevOpsProject "internal"
+#>
+function Find-JavaPackageWorkItem {
+  param (
+    [Parameter(Mandatory = $false)]
+    [string]$DevOpsOrganization = "azure-sdk",
+    [Parameter(Mandatory = $false)]
+    [bool]$IncludeClosed = $false
+  )
+
+  Write-Host "Fetching all Java package work items from DevOps..."
+  
+  # Build field list - same as FindPackageWorkItem
+  $fields = @()
+  $fields += "System.ID"
+  $fields += "System.State"
+  $fields += "System.AssignedTo"
+  $fields += "System.Parent"
+  $fields += "System.Tags"
+  $fields += "Custom.Language"
+  $fields += "Custom.Package"
+  $fields += "Custom.PackageDisplayName"
+  $fields += "System.Title"
+  $fields += "Custom.PackageType"
+  $fields += "Custom.PackageTypeNewLibrary"
+  $fields += "Custom.PackageVersionMajorMinor"
+  $fields += "Custom.PackageRepoPath"
+  $fields += "Custom.ServiceName"
+
+  $fieldList = ($fields | ForEach-Object { "[$_]"}) -join ", "
+  
+  # Build query - same as FindPackageWorkItem but without Package filter
+  $query = "SELECT ${fieldList} FROM WorkItems WHERE [Work Item Type] = 'Package'"
+  
+  if (!$IncludeClosed) {
+    $query += " AND [State] <> 'No Active Development' AND [PackageTypeNewLibrary] = true"
+  }
+  
+  # Filter by Java language only (no Package filter)
+  $query += " AND [Language] = 'Java'"
+  
+  # Exclude test items
+  $query += " AND [Tags] NOT CONTAINS 'Release Planner App Test'"
```

Comment:
@praveenkuttappan this is the script to update the existing work items for the Java package name. The query returns around 5000 work items. Is there any more filter we need to apply for the query?